examples/partitioner_examples/splitter_usage.py

```diff
@@ -15,13 +15,11 @@
 import spynnaker8 as p
 from pyNN.utility.plotting import Figure, Panel
 import matplotlib.pyplot as plt
-
-from pacman.model.partitioner_splitters import SplitterOneToOneLegacy
+from pacman.model.partitioner_splitters import (
+    SplitterOneToOneLegacy as OneToOneSplitter)
 from spynnaker.pyNN.extra_algorithms.splitter_components import (
-    SplitterAbstractPopulationVertexSlice)
-from spynnaker.pyNN.extra_algorithms.splitter_components.\
-    spynnaker_splitter_slice_legacy import (
-        SpynnakerSplitterSliceLegacy)
+    SplitterAbstractPopulationVertexSlice as APVSplitter,
```

Comment:
why rename? its not like you've saved any lines doing so

Reply:
reverted